R/FilteredData.R

```diff
@@ -759,6 +771,23 @@ FilteredData <- R6::R6Class( # nolint
             }
           )
 
+          observeEvent(
+            eventExpr = list(input[["filter_turn_onoff"]]),
+            handlerExpr = {
+              if (isFALSE(input[["filter_turn_onoff"]])) {
+                shinyjs::hide("filter_add_vars")
+                shinyjs::hide("filter_active_vars")
```

Comment:
shinyjs::disable instead of hide? I think that's nicer - though that would then mean you'd need some more logic `if input$switch-button` somewhere

Reply:
I thought of it, as you like it more (and I am neural) so I will apply it:)